.yarnrc.yml

```diff
@@ -1,10 +1,10 @@
+compressionLevel: mixed
+
+enableGlobalCache: false
```

Comment:
Should not be for CI only?

Reply:
As discussed, between the new default value set to true and the necessity to handle it in a user friendly way, let's do it in another user story